### DISCUSSION TITLE
the Font size not work in Entry

### DISCUSSION THREAD

**Discussion by akeyi2018** [israel-dryer/ttkbootstrap#515] [Q&A] [answered]
The code was under blow.
 ```py
import ttkbootstrap as ttk

# label
self.e_label = Label(text='category:', font=self.font)
self.e_label.pack()
# Entry
self.entry = ttk.Entry(bootstyle='info')
self.entry.pack()
self.style.configure('info.TEntry', font=('Helvetica', 24))
```
The font size was too small in Entry box. Is there anything I can do about it? 
<img width="145" alt="image" src="https://github.com/israel-dryer/ttkbootstrap/assets/34048522/e47ac2ea-eab3-4162-92c4-2c1d083d4474">

**Comment by israel-dryer** [comment, marked as answer]
This is a ttk quirk rather than a ttkbootstrap bug: **`ttk::entry` doesn't read its font from the style.** `style.configure('info.TEntry', font=('Helvetica', 24))` is accepted and stored (you can see it via `ttk::style lookup info.TEntry -font`), but the entry never renders it — so the text stays small. Unlike `TLabel`/`TButton`, the editable text widgets (`TEntry`, `TCombobox`, `TSpinbox`) carry `-font` as a **widget option** that defaults to the named font `TkTextFont`, and the widget always uses that instead of the style.

Two ways to actually change it:

**1. Set the font as a widget option** (simplest — per widget):

```python
self.entry = ttk.Entry(bootstyle='info', font=('Helvetica', 24))
# or after creation:
self.entry.configure(font=('Helvetica', 24))
```

**2. Reconfigure a named font** (one handle for many entries, updates live):

```python
from tkinter import font

# Option A: change the default font every text widget already uses
font.nametofont('TkTextFont').configure(size=24)

# Option B: define your own named font and attach it to the entry
my_font = font.Font(name='EntryFont', family='Helvetica', size=24)
self.entry = ttk.Entry(bootstyle='info', font='EntryFont')
# later, restyle every entry using it at once:
my_font.configure(size=18)
```

Because the entry holds a reference to the named font, mutating the font object reflows every widget using it — no need to touch each entry.

**The gotcha to avoid:** `self.style.configure('info.TEntry', font=...)` looks correct but is a no-op for the entry's rendered text. (If you saw it "work" alongside a `TkTextFont.configure(...)` call, the named-font change was doing the work, not the style.) Use the widget option or a named font instead.
